ui/components/MeshAdapterConfigComponent.js

```diff
@@ -63,110 +115,111 @@ const styles = (theme) => ({
     marginRight : theme.spacing(1),
     marginBottom : theme.spacing(1),
   }
-});
-
+}));
 const STATUS = {
   DEPLOYED : "DEPLOYED",
   UNDEPLOYED : "UNDEPLOYED",
   DEPLOYING : "DEPLOYING",
   UNDEPLOYING : "UNDEPLOYING",
 }
 
-class  MeshAdapterConfigComponent extends React.Component {
-  constructor(props) {
-    super(props);
-    const { meshAdapters } = props;
-    this.labelRef = React.createRef();
-    this.state = {
-      meshAdapters,
-      setAdapterURLs : [],
-      availableAdapters : [],
-      ts : new Date(),
-      meshLocationURLError : false,
-      selectedAvailableAdapterError : false,
-      adapterStates : {},
-    };
-  }
-
-  static getDerivedStateFromProps(props, state) {
-    const { meshAdapters, meshAdaptersts } = props;
-    // if(meshAdapters.sort().join(',') !== state.meshAdapters.sort().join(',')){
-    if (meshAdaptersts > state.ts) {
-      return {
-        meshAdapters,
-        ts : meshAdaptersts
-      };
+const MeshAdapterConfigComponent = (props) => {
+
+  const labelRef = useRef(null)
+  // this.state = {
```

Comment:
Same here we can directly remove these commented codes. Or specify why these are commented.